packages/vitest/src/node/reporters/dot.ts

```diff
@@ -70,10 +71,15 @@ export class DotReporter extends BaseReporter {
   }
 
   onTestCaseResult(test: TestCase): void {
+    const result = test.result().state
+    if (!this.isTTY && result !== 'pending') {
+      (this.ctx.logger.outputStream as Writable).write(formatTests([result]))
```

Comment:
This is breaking the newlines of other logging quite badly. See how Node's logging and Vitest's `stderr`/`stdout` test logs are shown here:

![image](https://github.com/user-attachments/assets/b4e1e719-b914-4886-804b-8b656537915a)

Instead it should look like this:

![image](https://github.com/user-attachments/assets/cc89f55d-f947-4eb5-b221-d41dad09ecd8)